src/app/api/socket/route.ts

```
@@ -0,0 +1,98 @@
+            { status: 400 },
+          );
+        sendToSocket(targetId, notification);
+        return NextResponse.json({ success: true, action });
+
+      case "sendToUser":
+        if (!targetId)
+          return NextResponse.json(
+            { success: false, error: "targetId required" },
+            { status: 400 },
+          );
+        const ok = sendToUser(targetId, notification);
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)